package/cloudshell/iac/terraform/services/clp_envvar_handler.py

```diff
@@ -75,11 +75,16 @@ def __init__(self, clp_res_model, clp_resource_attributes, shell_helper):
         self._shell_helper = shell_helper
 
     def set_env_vars_based_on_clp(self):
+        project_flag = False
+        cred_flag = False
         for attr in self._clp_resource_attributes:
-            if self.does_attribute_match(self._clp_res_model, attr, self._shell_helper, "CREDENTIALS JSON PATH"):
+            if self.does_attribute_match(self._clp_res_model, attr, "Google Cloud Provider.Credentials Json Path"):
                 os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = attr.Value
-            if self.does_attribute_match(self._clp_res_model, attr, self._shell_helper, "PROJECT"):
+                self._shell_helper.sandbox_messages.write_message(f"GOOGLE_APPLICATION_CREDENTIALS: {attr.Value}")
```

Comment:
it looks like something that was added for debugging purpose. we probably dont want to write this to the sandbox console,